.gitignore

```diff
@@ -1,2 +1,6 @@
 ./install/pythia8/
 ./install/python_packages/
+*.png
+*.pkl
+*.pdf
+
```

Comment:
let's also add `./models` here to avoid accidentally committing generated contents in the future